frontend/app/models/search.rb

```diff
@@ -13,7 +13,14 @@ def self.for_type(repo_id, type, criteria)
   def self.all(repo_id, criteria)
     build_filters(criteria)
 
+    puts 'here1'
```

Comment:
This looks like debugging output, should it maybe be dropped or replaced with a log statement, also apply to other `puts` in this method.

Reply:
I thought I got rid of those...